From 2b14b3c38210f8454894cec887915820a28b3ef8 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Thu, 30 Dec 2021 06:57:09 +0200
Subject: [PATCH] loosen requirements

* Remove unused libclang requirement
* Remove unecessary tensorflow-io-gcs-filesystem

Co-authored-by: Mark Harfouche <mark.harfouche@gmail.com>
Co-authored-by: ngam
---
 tensorflow/tools/pip_package/setup.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/tensorflow/tools/pip_package/setup.py b/tensorflow/tools/pip_package/setup.py
index face32bbb9a2e..6b251d8ac17ee 100644
--- a/tensorflow/tools/pip_package/setup.py
+++ b/tensorflow/tools/pip_package/setup.py
@@ -92,7 +92,6 @@ def standard_or_nightly(standard, nightly):
     # TODO(b/239052279): replace with external dependency on JAX repo once JAX
     # no longer relies on TF.
     'jax >= 0.3.15',
-    'libclang >= 13.0.0',
     # TODO(b/263178356): numpy 1.24 breaks TF's tests
     'numpy >= 1.22, <1.24',
     'opt_einsum >= 2.3.2',
@@ -112,8 +111,6 @@ def standard_or_nightly(standard, nightly):
     # TODO(b/266362323): wrapt==1.15.0rc0 incompatible with TF 2.12.0 RC0 (and
     # nightly, but works with TF 2.11)
     'wrapt >= 1.11.0, <1.15',
-    'tensorflow-io-gcs-filesystem >= 0.23.1;platform_machine!="arm64" or ' +
-    'platform_system!="Darwin"',
     # grpcio does not build correctly on big-endian machines due to lack of
     # BoringSSL support.
     # See https://github.com/tensorflow/tensorflow/issues/17882.

-- 
2.35.0
